compiler/qsc_frontend/src/compile.rs

```diff
@@ -123,6 +143,28 @@ impl SourceMap {
     pub fn iter(&self) -> impl Iterator<Item = &Source> {
         self.sources.iter()
     }
+
+    /// Returns the sources as an iter, but with the project root directory subtracted
+    /// from the individual source names.
+    pub(crate) fn localized_project_sources(&self) -> impl Iterator<Item = Source> + '_ {
```

Comment:
The word `localized` heavily connotes "localization" to me, is `relative` possibly a better term?